### DISCUSSION TITLE
续签Let's Encrypt证书 的自动任务，会把没有开启ssl的站点的域名自动续签，导致失败。

### DISCUSSION THREAD

**Discussion by helloworldiambeok** [midoks/mdserver-web#520] [General]
![Snipaste_2024-01-05_17-50-47](https://github.com/midoks/mdserver-web/assets/13003236/38e5cf5b-3ec0-4a4b-b6b9-f29304439e04)
如何单独的设置不需要自动续签的域名
或者设置不开启ssl的域名不参与自动续签。

**Comment by midoks** [comment]
已经解决，仅支持acme
